### DISCUSSION TITLE
Display active power sum L1+L2+L3 on Shelly 3EM running Tasmota

### DISCUSSION THREAD

**Discussion by rmenzing** [arendst/Tasmota#18994] [Support request]
I recently started using a Shell 3em with Tasmota. Now I would like to see the sum of active power L1+L2+L3 displayed in the web interface. How can I achieve this

**Comment by sfromis** [comment]
In general, do not see the Tasmota web UI as a device dashboard to get info from. It is mainly for admin/debug purposes, with the rest expected to be handled by your backend smarthome setup, encompassing multiple devices. Tasmota deliberately only have a very basic and rudimentary web UI, instead of it being the place to operate or follow the device.
